service/gradle/openapi.gradle

```diff
@@ -24,6 +24,7 @@ swaggerSources {
 // so it is available for the swagger_ui.html endpoint
 task copyOpenApiToResources(type: Copy) {
   description 'Copy the openapi results into build resources to support swagger_ui'
+  // TODO: declare openApiSource as a project output to avoid reaching into the other project from here.
```

Comment:
Add ticket?

Reply:
I'm sure I'll be back in here soon enough.